src/common/components/NFTCard/NFTCard.tsx

```diff
@@ -2,15 +2,22 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 import cn from 'classnames'
 
-export interface INFTCompactCardProps {
+import styles from './NFTCard.module.css'
+
+export type TNFTCompactCardProps = {
   imageSrc: string
   title: string
   likes: number
   liked: boolean
   className?: string
+  hideFollow?: boolean
+  hideLikeButton?: boolean
+  percentage?: number
+  variant?: string
+  isLastBid?: boolean
 }
 
-export interface INFTCardProps extends INFTCompactCardProps {
+export interface INFTCardProps extends TNFTCompactCardProps {
```

Comment:
Please update this as type.
import { Override } from '../../utils/types'
type TNFTCardProps = Override<
  TNFTCompactCardProps,
  { ... }
>

Please check inputs/inputNumberFormat